scrapy_autoextract/cache.py

```diff
@@ -88,3 +89,28 @@ def __setitem__(self, fingerprint: str, value) -> None:
 
     def close(self):
         self.db.close()
+
+
+class ScrapyCloudCollectionCache(_Cache):
+    def __init__(self, project, collection):
+        sc = ScrapinghubClient()
+        self.collection = sc.get_project(project).collections.get_store(collection)
+
+    @classmethod
+    def fingerprint(cls, request: Request) -> str:
+        return request.url
+
+    def __getitem__(self, fingerprint: str):
+        try:
+            return self.collection.get(fingerprint)
+        except NotFound:
+            raise KeyError
+
+    def __setitem__(self, fingerprint: str, value) -> None:
+        self.collection.set(
+            {'_key': fingerprint,
+             'value': value}
+        )
+
+    def close(self):
+        pass
```

Comment:
Shouldn't be closing the `sc` client here?

Reply:
That's right. Thanks!